src/BingAIClient.js

```diff
@@ -5,15 +5,18 @@ import Keyv from 'keyv';
 
 export default class BingAIClient {
     constructor(opts) {
-        this.opts = opts;
+        this.opts = {
+            ...opts,
+            host:opts.host||'www.bing.com'
```

Comment:
```suggestion
            host: opts.host || 'https://www.bing.com',
```

Reply:
I think this will be better so that if you want to run a local reverse proxy or something, this will support that.